src/components/_/TipTap.vue

```diff
@@ -134,6 +134,33 @@
           <IconMinus />
         </ButtonIcon>
       </div>
+      <div class="flex gap-1">
+        <ButtonIcon
+          :aria-label="t('formatAlignLeft')"
+          :title="t('formatAlignLeft')"
+          @click="editor?.chain().focus().setTextAlign('left').run()"
+        >
+          <IconLeftAlignment />
+        </ButtonIcon>
+      </div>
+      <div class="flex gap-1">
+        <ButtonIcon
+          :aria-label="t('formatAlignCenter')"
+          :title="t('formatAlignCenter')"
+          @click="editor?.chain().focus().setTextAlign('center').run()"
+        >
+          <IconCenterAlignment />
+        </ButtonIcon>
+      </div>
+      <div class="flex gap-1">
```

Comment:
The buttons belong to the same group, so they do not all need to be wrapped with `gap-1`.

Reply:
done